lib/compress/index.js

```diff
@@ -3982,7 +3982,7 @@ merge(Compressor.prototype, {
     AST_Scope.DEFMETHOD("var_names", function varNames() {
         var var_names = this._var_names;
         if (!var_names) {
-            this._var_names = var_names = this.parent_scope instanceof AST_Defun ? varNames.call(this.parent_scope) : new Set();
```

Comment:
Let's hope it's the last time we have to change this line in 2019 lol

Reply:
I hope too :)